### DISCUSSION TITLE
part-ii/2021-16/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#409] [Question comments]
Part II 2021 Applied Probability: Paper 1, Section II, 28K 

https://questions.tripos.org/part-ii/2021-16/

**Comment by ArthurConmy** [comment]
<details>
<summary>b)</summary>

I get a $N(0,z)$ random variable, by using convergence of characteristic function.
</details> 

<details>
<summary>c)</summary>

I get mean $z\int_S g(y) dy$ and variance $z \int_S [g(y)]^2 dy$.
</details>
